src/Event/NewConferencesEvent.php

```diff
@@ -16,13 +16,9 @@
 
 class NewConferencesEvent extends Event
 {
-    /** @var array<Conference> */
```

Comment:
We have lost this information. Is there a way to keep it ? (same with in src/Conferences/ConferencesHarvester.php and in other files)